models/restaurant.js

```diff
@@ -47,23 +53,25 @@ class Restaurant {
         //console.log("location:", location, city, postal_code, loc, restaurantName, OM_API_KEY)
         const result = await axios.get(`https://openmenu.com/api/v2/location.php?key=${OM_API_KEY}&country=us&${location}=${loc}&s=${restaurantName}`)
             .catch((err) => {
-                //console.log(err)
+                console.log(err)
             })
         //console.log(result.data.response.result.restaurants[0].id)
         return result.data.response.result.restaurants[0].id
     }
 
+    //Calls the API and returns details about a restaurant
     static async apiRestaurantInfo(id) {
         //console.log("in apiRestaurantInfo: id:", id)
         const result = await axios.get(`https://openmenu.com/api/v2/restaurant.php?key=${OM_API_KEY}&id=${id}`)
             .catch((err) => {
-                //console.log(err)
+                console.log(err)
```

Comment:
REQUIRED: Remove Remove console logs in your commits.

Reply:
Removed